### PR TITLE
Improve description of the HTTP JSON API in the Ledger API docs

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
changelog_begin
changelog_end

As a way to exemplify https://github.com/digital-asset/daml/issues/13029#issuecomment-1061570459

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/app-dev/ledger-api.rst [hunk 1]
```diff
@@ -23,8 +23,13 @@ another language. Every ledger that Daml can run on exposes this same API.
 What's in the Ledger API
 ************************
 
-You can access the Ledger API via the HTTP JSON API, Java bindings or gRPC. In
-all cases, the Ledger API exposes the same services:
+You can access the Ledger API via the Java bindings.
+
+If you don't use a language that target the JVM you can use gRPC to generate the code to access the Ledger API in
```
Comment (carrielaben-da) [inline docs/source/app-dev/ledger-api.rst:28]: > "If you don't use a language that target the JVM"

"If you don't use a language that targets the JVM"

File: docs/source/app-dev/ledger-api.rst [hunk 2]
```diff
@@ -52,6 +57,11 @@ For full information on the services see :doc:`/app-dev/services`.
 
 You may also want to read the :doc:`protobuf documentation </app-dev/grpc/proto-docs>`, which explains how each service is defined as protobuf messages.
 
+As an alternative way to tap into the Ledger API you can also use the HTTP JSON API.
```
Comment (carrielaben-da) [inline docs/source/app-dev/ledger-api.rst:60]: "As an alternative way to tap into the Ledger API you can also use the HTTP JSON API." --> "You can also used the HTTP JSON API to tap into the Ledger API."

Apologies, the suggest feature doesn't seem to be available here.

File: docs/source/app-dev/ledger-api.rst [hunk 3]
```diff
@@ -52,6 +57,11 @@ For full information on the services see :doc:`/app-dev/services`.
...
+At its core, it provides a simplified view of the active contract set, exposing it via a conventional, well-defined
+JSON-based encoding over a conventional HTTP connection. On top of that, a subset of the services mentioned above is
```
Comment (carrielaben-da) [inline docs/source/app-dev/ledger-api.rst:62]: We can probably just delete "On top of that".

**Comment by stefanobaghino-da** [comment]
Closing in favor of https://github.com/digital-asset/daml/pull/13271.

@carrielaben-da thanks for your comments, I addressed them in the PR linked in this comment. 🙇🏻‍♂️

**Pull request closed without merging**